src/pages/Login/Login.test.tsx

```diff
@@ -57,13 +183,56 @@ describe("Clicking go to authenticate button", () => {
       </Router>
     );
 
-    userEvent.click(container.querySelector("ion-button") as TargetElement);
+    const phoneInput = "12345678";
+    fillInput(container, phoneInput);
+    clickButton(container);
 
+    await wait(() => expect(mockHistoryPush).toHaveBeenCalledTimes(1));
     expect(mockHistoryPush).toHaveBeenCalledWith("/authenticate", {
       from: "/offers",
     });
-    expect(mockHistoryPush).toHaveBeenCalledTimes(1);
 
+    mockHistoryPush.mockRestore();
     mockUseLocation.mockRestore();
   });
 });
+
+describe("On receiving error response from send_pin", () => {
+  it("should log the error to the console", async () => {
```

Comment:
`console.log` isn't really an error management, we probably want an error shown to the user